submit-web/src/components/SubmissionItem/ManagementPlanSubmission/index.tsx

```diff
@@ -156,6 +167,8 @@ export const ManagementPlanSubmission = () => {
     callSaveSubmission({
       data: {
         type: SUBMISSION_TYPE.FORM,
+        status: status || SUBMISSION_STATUS.COMPLETED.value,
```

Comment:
we should never default in this case, we should never let the status be undefined